sqlx-core/Cargo.toml

```diff
@@ -62,6 +62,7 @@ all-types = [
 bigdecimal = ["bigdecimal_", "num-bigint"]
 decimal = ["rust_decimal", "num-bigint"]
 json = ["serde", "serde_json"]
+ltree = ["postgres"]
```

Comment:
Drive-by review: this might not require a feature if it doesn't depend on any external dependency

Reply:
Well yes, it was the idea as its an extension

Reply:
Removed the feature